zombie-tests/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "zombie-tests",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {},
+  "dependencies": {
+    "@polkadot-api/cli": "workspace:*",
+    "@polkadot-api/client": "workspace:*",
+    "@polkadot-api/sc-provider": "workspace:*",
+    "@polkadot-api/substrate-bindings": "workspace:*",
+    "@polkadot-api/substrate-client": "workspace:*",
+    "@polkadot-api/substrate-codegen": "workspace:*",
+    "@polkadot-api/utils": "workspace:*",
+    "typescript": "^5.1.6"
```

Comment:
check if `typescript` can be removed so it fallback to the one specified at the root